sale_commission_delegate_partner/models/sale_commission_settlement.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class SaleCommissionSettlement(models.Model):
+    _inherit = 'sale.commission.settlement'
+
+    def _get_settlement_invoice_partner(self):
```

Comment:
Remove this

Reply:
If I make the change on `_prepare_invoice_header` I need to do again the new and the onchange and I thought it was better to avoid that. WDYT?